examples/ios/swift/ListSwiftUI/Views/ContentView.swift

```diff
@@ -268,9 +268,7 @@ struct ContentView: View {
     var body: some View {
         NavigationView {
             VStack {
-                if #available(iOS 15.0, macOS 12.0, tvOS 15.0, watchOS 8.0, *) {
-                    // Don't add a SearchView in case searchable is available
-                } else {
+                if #unavailable(iOS 15.0, macOS 12.0, tvOS 15.0, watchOS 8.0) {
```

Comment:
#unavailable was added in Swift 5.6 and our minimum supported version is 5.5.

Reply:
I disable the swift lint error while we are on swift 5.5 minimum version.